### ISSUE TITLE
kw_if isssue

### ISSUE THREAD

**Issue by JanPoonthong** [quick-lint/quick-lint-js#88]
This is my code 
```
document.addEventListener('DOMContentLoaded', () => {
    document.querySelector('select').onchange = function() {
        document.querySelector('#hello').style.color = this.value;
    }
});
asjdfhjksdahfj
if :
kjaslkjd;kjaslkjd
```

when i run quick-lint-js, i get 
```
/home/jan/tmp/quick-lint-js/src/parse.cpp:602: fatal: token not implemented in parse_expression_remainder: kw_if on line 8 column 1
zsh: illegal hardware instruction  /home/jan/tmp/quick-lint-js/build/quick-lint-js
```

**Comment by strager** [comment]
Crashes fixed by commits c49dfdc180252896d503a702de3940cad5783da5, e0525e46a1ed52842d4381f785e7f13d3771a6f3, and 25169123a35c3e49c198c8ca195f9c09d6de5f1f.

**Comment by JanPoonthong** [comment]
Perfect!
